src/ert/config/observations.py

```diff
@@ -123,7 +123,8 @@ def _handle_history_observation(
         if history_type is None:
             raise ValueError("Need a history type in order to use history observations")
 
-        response_config.keys.append(summary_key)
+        if summary_key not in response_config.keys:
+            response_config.keys.append(summary_key)
```

Comment:
Not necessarily wrong, but I don't understand why this is needed.

Reply:
It is quite common to have multiple observations pointing to the same response key, and we then get a bunch of duplicates without this.